Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm

```diff
@@ -231,13 +231,40 @@ static bool experimentalFeatureEnabled(const String& key, bool defaultValue = fa
     WebCore::removeRecentlyModifiedRecentSearches(oldestTimeToRemove);
 }
 
-NSString *WebDatabaseDirectoryDefaultsKey = @"WebDatabaseDirectory";
-NSString *WebStorageDirectoryDefaultsKey = @"WebKitLocalStorageDatabasePathPreferenceKey";
-NSString *WebKitMediaCacheDirectoryDefaultsKey = @"WebKitMediaCacheDirectory";
-NSString *WebKitMediaKeysStorageDirectoryDefaultsKey = @"WebKitMediaKeysStorageDirectory";
+String WebsiteDataStore::defaultWebsiteDataStoreDirectory(const String& identifier)
+{
+    static dispatch_once_t onceToken;
+    static NeverDestroyed<RetainPtr<NSURL>> websiteDataStoreDirectory;
+    dispatch_once(&onceToken, ^{
+        NSURL *libraryDirectory = [[NSFileManager defaultManager] URLForDirectory:NSLibraryDirectory inDomain:NSUserDomainMask appropriateForURL:nullptr create:NO error:nullptr];
```

Comment:
I'm not sure if NSLibraryDirectory is where we want all the data, but seems reasonable.

Reply:
We've been using Library directory for default website data directory (see `WebsiteDataStore::websiteDataDirectoryFileSystemRepresentation`)